docs/configuration/scenarios/scenario-000.md

```diff
@@ -0,0 +1,11 @@
+# Setting the reminder period
+
+[#38](https://github.com/team-work-tools/team-work-telegram-bot/issues/38)
+
+## Description
+
+The command /set_reminder_period *time in minutes* will set the time for the user after which, if the user has not answered all the questions, he/she will receive a reminder in private messages to answer them until he/she has answered all questions in the chat.
+
+[Ts code for a diagram using the XState extension in VS Code](https://github.com/team-work-tools/team-work-telegram-bot/blob/main/docs/configuration/snippets/snippet-000.ts)
```

Comment:
Use the relative path `docs/configuration/snippets/snippet-000.ts` instead of the full URL `https://github.com/team-work-tools/team-work-telegram-bot/blob/main/docs/configuration/snippets/snippet-000.ts`. GitHub automatically converts links with paths to URLs.